portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,33 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+function randomRecipe() {
+  // there are three recipes total
+  const imageIndex = Math.floor(Math.random() * 4) + 1;
+  const imgUrl = "/images/recipes/" + imageIndex + ".jpg";
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const imgElement = document.createElement("img");
+  imgElement.src = imgUrl;
+  const imageContainer = document.getElementById("random-image-container");
+  
+  if(imageIndex == 1){
```

Comment:
rather than having a bunch of if statements, you could make the code cleaner by making use of maps (image index -> html) or (image index -> title -> link), etc. etc. lots of ways to do it.

Reply:
i'm currently working on this, but as of now, i pushed what i had originally so i can move onto merging & deploying

Reply:
sg, maybe add a TODO then